superglue/lib/utils/request.ts

```diff
@@ -66,32 +66,23 @@ export function argsForFetch(
   { method = 'GET', headers = {}, body = '', signal }: RequestInit = {}
 ): [string, RequestInit] {
   method = method.toUpperCase()
-
   const currentState = getState().superglue
 
-  const jsAccept = 'application/json'
-  headers = {
-    ...headers,
-    accept: jsAccept,
-    'x-requested-with': 'XMLHttpRequest',
-    'x-superglue-request': 'true',
-  }
-
-  // This needs to be done better. This is saying to
-  // remove the content-type header from UJS form
-  // submissions.
-  const fromUJSForm = headers['content-type'] === null
```

Comment:
Found a better way to todo this. If the body is FormData, we just remove the `content-type` header. This also changes the tests down below where you'll see removal of the header